queries/tags.scm

```diff
@@ -0,0 +1,21 @@
+(binary_operator
+    lhs: (identifier) @name
+    operator: "<-"
+    rhs: (function_definition)
+) @definition.function
+
+(binary_operator
+    lhs: (identifier) @name
+    operator: "="
+    rhs: (function_definition)
+) @definition.function
+
+(call
+    function: (identifier) @name
+) @reference.call
+
+(call
+    function: (namespace_operator
```

Comment:
Is there any useful highlight we could apply to package names here?

Reply:
See original paragraph about `reference.module` https://github.com/r-lib/tree-sitter-r/pull/97#issue-2260037121

TLDR is that I don't think there is anything we can tag them with that github search would use (this isn't highlighting, this is for jump to definition and search)